Makefile

```
@@ -151,6 +151,9 @@ MANIFEST_ROOT ?= config
 CRD_ROOT ?= $(MANIFEST_ROOT)/crd/bases
 WEBHOOK_ROOT ?= $(MANIFEST_ROOT)/webhook
 RBAC_ROOT ?= $(MANIFEST_ROOT)/rbac
+ASO_CRDS_PATH := $(MANIFEST_ROOT)/aso/crds.yaml
+ASO_VERSION := v2.0.0
```

We released v2.1.0 of ASO recently.

```suggestion
ASO_VERSION := v2.1.0
```

---

Yep, there's a dependabot PR open (#3605) with the go.mod/go.sum changes and I was thinking of adding that change there. Just trying to keep things tidy and hopefully easier to review.